.vscode/launch.json

```diff
@@ -23,14 +12,30 @@
             "program": "${workspaceFolder}/cmd/porch/main.go",
             "args": [
                 "--secure-port=4443",
-                "--v=7",
-                "--standalone-debug-mode",
+                // "--v=7",
+                // "--standalone-debug-mode",
                 "--kubeconfig=${env:KUBECONFIG}",
-                "--cache-directory=${workspaceFolder}/.cache",
                 "--function-runner=172.18.255.201:9445",
-                "--repo-sync-frequency=60s"
+                "--repo-sync-frequency=60s",
```

Comment:
Is jsonlint ok with this one?

Reply:
you are right, this is not nice. Fixing it.